crates/provider/src/builder.rs

```diff
@@ -347,6 +346,11 @@ impl<L, F, N> ProviderBuilder<L, F, N> {
     }
 }
 
+type JoinedEthereumWalletFiller<F> = JoinFill<F, WalletFiller<alloy_network::EthereumWallet>>;
+
+#[cfg(any(test, feature = "anvil-node"))]
+type AnvilProviderResult<T> = Result<T, alloy_node_bindings::anvil::AnvilError>;
```

Comment:
Added these type aliases to avoid clippy error

Reply:
@mattsse By the way, should we create a type alias for `Http<reqwest::Client>`? It appears so many times in this repository and it can cause clippy warnings `very complex type used. Consider factoring parts into type definitions` when a type is complex, as in this case.